### PR TITLE
Elastic 5.3

### LINKED ISSUE #825: etcd 3.1.4 fails to start in the amplifier stack

### PR DESCRIPTION
**Pull request by ndegory**
# content

- update elasticsearch version to 5.3.0
- new image for kibana (5.3.0)
- fix #825 (etcd 3.1.5)

# how to test

```
ampmake clean build build-cli-wrapper
amp start
for image in amp-agent ampbeat amplifier; do docker tag appcelerator/${image}:local localhost:5000/appcelerator/${image}:local ; docker push localhost:5000/appcelerator/${image}:local ; done
docker run -it --rm --network=hostnet -v $PWD/stacks:/stacks docker --host=m1 stack deploy -c /stacks/monitoring.stack.yml monitoring
docker run -it --rm --network=hostnet -v $PWD/stacks:/stacks docker --host=m1 stack deploy -c /stacks/amplifier.stack.yml amplifier
docker exec m1 docker service ls # amplifier, etcd, elasticsearch and kibana should be up
# you should be able to connect to http://localhost:5601
amp stop
```

### CODE REVIEW

**Review by freignat91 — APPROVED** [review]
LGTM

**Comment by subfuzion** [comment]
@ndegory Please rebase again.

File: images/kibana/docker-compose.production.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: "2"
```
Comment (subfuzion) [inline images/kibana/docker-compose.production.yml]: What are the compose v2 files for? Tests?

**Review by subfuzion — CHANGES_REQUESTED** [review]

File: images/kibana/docker-compose.production.yml (hunk 1 above)
Comment (subfuzion) [inline images/kibana/docker-compose.production.yml]: I mean to ask "What are the compose v2 files for?" We don't need them...

Comment (ndegory) [reply images/kibana/docker-compose.production.yml]: these are unit tests for the images, which were used with automated builds on Docker Hub, and are now used by the images/build-images.sh script. During development phases testing the images only through the AMP core stack would be a lot of pain.

Comment (subfuzion) [reply images/kibana/docker-compose.production.yml]: Having `production` as part of the filename is misleading. If this is just for testing then why not consolidate them into a single `.test.yml`?

**Review by subfuzion — APPROVED** [review]
LGTM

**Comment by subfuzion** [comment]
The Travis CI build passed, although there seems to be an issue with the status being reported for the CI check that will need to be investigated. However, this shouldn't hold up this PR, so disregarding CI check for now and performing an administrator merge.

**Pull request merged**